examples/python/client.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import grpc
+import argparse
+from armonik.client import ArmoniKSubmitter
+from armonik.common import TaskDefinition, TaskOptions
+from datetime import timedelta
+from common import Payload, Result
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser("Armonik Example Client")
+    parser.add_argument("-e", "--endpoint", required=True, type=str, help="Control plane endpoint")
+    parser.add_argument("-p", "--partition", type=str, help="Partition used for the worker")
+    parser.add_argument("-v", "--values", type=float, help="List of values to compute instead of x in [0, n[", nargs='+')
+    parser.add_argument("-n", "--nfirst", type=int, help="Compute from 0 inclusive to n exclusive, n=10 by default", default=10)
+    return parser.parse_args()
+
+
+def main():
+    args = parse_arguments()
+    # Open a channel to the control plane
+    with grpc.insecure_channel(args.endpoint) as channel:
+        # Create a task submitting client
+        client = ArmoniKSubmitter(channel)
+        # Default task options to be used in a session
+        default_task_options = TaskOptions(max_duration=timedelta(seconds=300), priority=1, max_retries=5)
+        # Create a session
+        session_id = client.create_session(default_task_options=default_task_options, partition_ids=[args.partition] if args.partition is not None else None)
+        try:
+            # Create the payload
+            payload = Payload([i for i in range(args.nfirst)] if args.values is None else args.values)
```

Comment:
I am suspecting that the ```]``` is not at the correct position.

Reply:
No it's right where it should be. Either you set the number N and generate from 0 to N, or you give it values directly